### ISSUE TITLE
Weird Smarty error message

### ISSUE THREAD

**Issue by musicpanda** [thirtybees/thirtybees#1256]
While experimenting with changes to the template code for the Minic Slider I suddenly got an error page where the with of things was seriously reduced. See attached image. I have also attached the template file (zipped) that caused this issue. 

I tried to get in the browser the source code of the error page. But the effort ended in an SBOX_FATAL_MEMORY_EXCEEDED error.
![smarty](https://user-images.githubusercontent.com/13930107/93352994-b6b07d00-f83b-11ea-8057-334373c09a3f.jpg)

[slides.zip](https://github.com/thirtybees/thirtybees/files/5232869/slides.zip)

**Comment by Dh42** [comment]
The error looks to be caused by line 19 actually since that variable is not closed.

**Comment by musicpanda** [comment]
>> The error looks to be caused by line 19 actually since that variable is not closed.

I understand that. My error report is not about that error but about the way it is reported - with the yellow rectangle reduced to a small bar.

**Comment by getdatakick** [comment]
From the screenshot it's obvious that you are using old(er) version of thirtybees -- error reporting has been redesign, and the error page should not look like this anymore.

Could you test if the issue is present on the latest version?

**Comment by musicpanda** [comment]
> Could you test if the issue is present on the latest version?

I did so and indeed the error page became normal. Unfortunately I found the experience rather stressful:
 - Core Updater is far from intuitive.
 - My CMS page in the backoffice now gives a 500 error. And it seems impossible to get a decent error message. Debug mode (_PS_MODE_DEV_) has no result. And there are no error messages: neither in the log directory of the shop nor in the Apache error log.
